core/templates/dev/head/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardBackendApiService.js

```diff
@@ -16,15 +16,27 @@
  * @fileoverview Service to retrieve information of topics and skills dashboard
   from the backend.
  */
+ oppia.constant(
+   'MERGE_SKILL_URL', '/merge_skill');
 
 oppia.factory('TopicsAndSkillsDashboardBackendApiService', [
-  '$http', function($http) {
+  '$http', 'MERGE_SKILL_URL', function($http, MERGE_SKILL_URL) {
     var _fetchDashboardData = function() {
       return $http.get('/topics_and_skills_dashboard/data');
     };
 
+    var _mergeSkills = function(old_skill_id, new_skill_id) {
```

Comment:
These args should be lowerCamelCased: oldSkillId, newSkillId. (You also don't need to change the keys in mergeSkillData below, but the values should be changed accordingly.)

Reply:
Done